keyvault/client/src/main/java/com/azure/keyvault/RequestIdPolicy.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I'm not sure if this is necessary. from the KeyVault REST API, it doesn't appear these headers are supported?
https://docs.microsoft.com/en-us/azure/key-vault/common-parameters-and-headers

Reply:
This policy is not needed.
Removed it.